jcli/src/jcli_app/vote/mod.rs

```diff
@@ -31,6 +32,14 @@ pub enum Error {
     },
     #[error("invalid committee member index")]
     InvalidCommitteMemberIndex,
+    #[error("failed to read encrypted tally bytes")]
+    EncryptedTallyRead,
+    #[error("failed to read decryption key bytes")]
+    DecryptionKeyRead,
+    #[error(transparent)]
+    FormatError(#[from] crate::jcli_app::utils::output_format::Error),
+    #[error(transparent)]
+    JsonError(#[from] serde_json::Error),
```

Comment:
This and other variants like `Io`, and `Hex` make the code easier to write, but will probably provide poor UX because they don't really tell which argument is at fault or where the problem has occurred.